src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1086,6 +1086,8 @@ protected virtual SecurityKey ResolveTokenDecryptionKey(string token, JsonWebTok
         /// <exception cref="ArgumentException">'token.Length' is greater than <see cref="TokenHandler.MaximumTokenSizeInBytes"/>.</exception>
         /// <remarks><para>If the <paramref name="token"/> is in JWE Compact Serialization format, only the protected header will be deserialized.</para>
         /// This method is unable to decrypt the payload. Use <see cref="ValidateToken(string, TokenValidationParameters)"/>to obtain the payload.</remarks>
+        /// <remarks>The token is NOT validated and no security decisions should be made about the contents.
```

Comment:
is it to spec to have multiple remarks method headers? I would think these should be merged into one remark.